packages/Webkul/CartRule/src/Http/Controllers/CartRuleController.php

```diff
@@ -34,7 +34,7 @@ public function __construct(protected CartRuleRepository $cartRuleRepository)
     /**
      * Display a listing of the resource.
      *
-     * @return \Illuminate\Http\Response
+     * @return \Illuminate\Contracts\Foundation\Application|\Illuminate\Contracts\View\Factory|\Illuminate\Contracts\View\View
```

Comment:
@return \Illuminate\Http\Response|\Illuminate\View\View

Reply:
`index` function return `Factory` OR `View` because use the `view` helper!
Also if the request is ajax return the object!